src/awkward/_v2/operations/ak_from_rdataframe.py

```diff
@@ -0,0 +1,40 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import awkward as ak
+
+
+def from_rdataframe(data_frame, column, column_as_record=True):
+    """
+    Args:
+        data_frame (`ROOT.RDataFrame`): ROOT RDataFrame to convert into an
+             Awkward Array.
+         column (str): A column to be converted to Awkward Array.
+         column_as_records (bool): If True, the column converted as records.
+
+     Converts ROOT Data Frame columns into an Awkward Array.
+
+     See also #ak.to_rdataframe.
+    """
+    with ak._v2._util.OperationErrorContext(
+        "ak._v2.from_rdataframe",
+        dict(column=column),
```

Comment:
```suggestion
        dict(
            data_frame=data_frame,
            column=column,
        ),
```

Needs all of the arguments (and I think the `column_as_record` is going away, right?).

Reply:
done